gapic/api/artman_longrunning.yaml

```diff
@@ -1,5 +1,7 @@
 common:
-  api_name: google-longrunning-v1
+  short_name: longrunning
+  version: v1
+  is_cloud_api: False
```

Comment:
related to my comment on artman - I think it would be better to have `org_id: google_cloud`